app/assets/javascripts/components/stories/Search.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Stories from './Stories';
+import * as Story from '../../models/beta/story';
+
+const propTypes = {
+  stories: PropTypes.array.isRequired
+};
+
+const defaultProps = {
+  stories: []
+};
+
+const Header = ({ stories }) =>
+  <div className="Sprint__header">
+    Stories found: {stories.length}
+
+    <div>
+      <span className="done-points">Points: {Story.totalPoints(stories)}</span>
```

Comment:
Should we have a custom class for that? Like `Sprint__done-points`. If we don't have it yet (like, if we're using `done-points` in other places in the v2) we could create it now and use it in both places of the v2 (here and the other ones)